manifests/server/export.pp

```diff
@@ -47,6 +47,10 @@
 # @param v4_export_name
 # @param nfsv4_bindmount_enable
 #
+# @param create_dir
+#   Boolean.  Create the directory to be exported.
+#   Defaults to true.
```

Comment:
same feedback as the other parameter with generated docs.

Reply:
Changed per your recommendations.  Thanks.